x/auth/vesting/types/vesting_account.go

```diff
@@ -199,14 +201,30 @@ type vestingAccountPretty struct {
 	VestingPeriods Periods `json:"vesting_periods,omitempty" yaml:"vesting_periods,omitempty"`
 }
 
+type vestingAccountJSON struct {
+	Address          sdk.AccAddress `json:"address" yaml:"address"`
+	Coins            sdk.Coins      `json:"coins" yaml:"coins"`
+	PubKey           crypto.PubKey  `json:"public_key" yaml:"public_key"`
```

Comment:
So this is the key change. Keep this type

Reply:
Yes, keep the type but not use bech32.